lib/SILOptimizer/IPO/GlobalOpt.cpp

```diff
@@ -808,6 +857,50 @@ static bool canBeChangedExternally(SILGlobalVariable *SILG) {
   return true;
 }
 
+static bool canBeUsedOrChangedExternally(SILGlobalVariable *global) {
+  if (global->isLet())
```

Comment:
When does this produce a different result in the isLet() case?

Reply:
Looking at it, I think they should always be the same but this way it will always match [the check here](https://github.com/apple/swift/blob/eb39989f10c1ca70e0542c65230238c1a7a07834/lib/SILOptimizer/IPO/GlobalOpt.cpp#L932-L937).